pkgs/top-level/qt5-packages.nix

```diff
@@ -51,9 +51,17 @@ let
     };
   in (lib.makeOverridable mkPlamoGear attrs);
 
-in (kdeFrameworks // plasmaMobileGear // plasma5 // plasma5.thirdParty // kdeGear // qt5 // {
+  mauiPackages = let
```

Comment:
Should this be called `mauiApps` because that's kinda what upstream does, even though we include the libraries.
Or should we even separate `mauiFramework` and `mauiApps`?

Reply:
Are they released together or separately?

Reply:
Together. They are all on version 2.1.2 right now.